platformLibs/src/platform/mingw/winscard.def

```diff
@@ -0,0 +1,15 @@
+package = platform.winscard
```

Comment:
These _all_ should be `platform.windows.winscard` or similar.

Despite the name, `winscard.h` exists on non-Windows platforms, and it has [subtle differences between Linux (which uses pcsclite) and macOS (which is based on a modified version of pcsclite) APIs](https://pcsclite.apdu.fr/api/group__API.html#details).

[It needs quite significant hand-holding to fix that...](https://github.com/micolous/kotlin-pcsc/)

Reply:
I'm not sure.
There always can be subtle differnces, even on `mingw_x86` (32-bit) and `mingw_x64` (64-bit) platforms.